cirq-rigetti/cirq_rigetti/service_test.py

```diff
@@ -13,21 +12,11 @@ def test_get_rigetti_qcs_service():
     assert service._quantum_computer.name == '9q-square-qvm'
 
 
-@pytest.mark.rigetti_integration
-def test_rigetti_qcs_service_api_call():
-    """test that `RigettiQCSService` will use a custom defined client when the
-    user specifies one to make an API call."""
-
-    class Response(httpx.Response):
-        def iter_bytes(self, chunk_size: Optional[int] = None) -> Iterator[bytes]:
-            yield b"{\"quantumProcessors\": [{\"id\": \"Aspen-8\"}]}"  # pragma: no cover
-
-    class Transport(httpx.BaseTransport):
-        def handle_request(self, request: httpx.Request) -> httpx.Response:
-            return Response(200)
+@patch('cirq_rigetti.service.QCSClient')
```

Comment:
This should exclude `test_list_quantum_processors` from testing on Mac and Windows platforms which do not start `cirq-rigetti/docker-compose.test.yaml`

```suggestion
@pytest.mark.rigetti_integration
@patch('cirq_rigetti.service.QCSClient')
```